daprdocs/data/components/state_stores/generic.yaml

```diff
@@ -11,7 +11,7 @@
     query: false
 - component: Apache Cassandra
   link: setup-cassandra
-  state: Alpha
+  state: Beta
   version: v1
   since: "1.0"
```

Comment:
@berndverst do you need to update the "since" fields too?

Reply:
The believe the since is supposed to indicate when this component was introduced and does not reflect changes in the state. If it is then then are others that we have not done this with, when they move to Beta.